### ISSUE TITLE
Toggle breakpoints should respect multiple cursors

### ISSUE THREAD

**Issue by asdf23** [microsoft/vscode#77879]
Issue Type: <b>Bug</b>

If I set the cursor on multiple lines (Highlight "throw" ,  press Ctrl + D, with the Sublime Keystrokes selected, this has the effect of highlighting the matching text on multiple lines)  the set a Breakpoint (F9) I only get a single Breakpoint set. My expectation is that at each highlighted line the Breakpoint is set.

VS Code version: Code 1.36.1 (2213894ea0415ee8c85c5eea0d0ff81ecc191529, 2019-07-08T22:55:08.091Z)
OS version: Linux x64 5.1.18-200.fc29.x86_64

<details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
csharp|ms-|1.21.0
sqltools|mtx|0.19.6

</details>

**Comment by isidorn** [comment]
This is a fair feature request and it makes sense to me. Assigning it to on-deck - so I will look into it once I find time.
Also adding help wanted. Myabe somebody is interested in providing a PR to tackle this.
Code pointer https://github.com/Microsoft/vscode/blob/cf03ea3729dc37e439d724267ad29dc5840f903a/src/vs/workbench/contrib/debug/browser/debugEditorActions.ts#L38

**Comment by mayaswrath** [comment]
I took a stab at this. First time looking at the vscode source and I have a couple questions. Work is here mayaswrath/vscode@4c2a3af2c8026002388910dc208ea284e05fddbc. I used `editor._getCursors()` to get the positions of the cursors. This function is commented as internal, so I am not sure if I should be using it. I am used to the _ prefix notation as being internal to a class. I see no other way to get the position of all the cursors, so if this is not allowed, seems like a bigger task. 

I assumed a test should go with this but there are no existing test for the debug editor actions. I didnt really see any tests for any editor actions (just the Line Operations test, but those only require an editor so are self contained). I see no easy way to create a DebugService, most of its dependencies don't have a mock. Seems like a smoke test would be pretty easy but I wasnt sure if that is the correct place for something like this. The smoke tests seemed pretty lean. What kind of tests are expected for a PR on something like this.

Also I have never done a public PR before, I assumed that should be done once its complete, hence why I linked to my fork. Is that proper procedure, or should I have just made a PR.
